Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2343,7 +2349,7 @@ extension BrowserViewController {
         _ = Try(withTry: {
             inputAssistant.trailingBarButtonGroups.last?.barButtonItems.append(item)
         }) { exception in
-            log.error("Failed adding custom search button to input assistant: \(String(describing: exception))")
+            log.error("Failed adding custom search button to input assistant: \(exception?.description ?? "nil")")
```

Comment:
Oh, this is hideous. Sorry, haha. I didn't realize `String(describing:)` took an optional, or that `foo?.description` would throw a dumb warning. I think older one is better 😬 ...

Reply:
😂

FINE